Extensions/TiledSpriteObject/Extension.cpp

```diff
@@ -135,6 +135,21 @@ void DeclareTiledSpriteObjectExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("GetHeight")
       .SetIncludeFile("TiledSpriteObject/TiledSpriteObject.h");
 
+  obj.AddAction("Size",
```

Comment:
Action should be named `SetXxx` (that's not enforced in the whole codebase, but I try to use this now for all actions).
Conditions and expressions can be named `Xxx` (without "Get").